### PR TITLE
shapes-arc

### PR DESCRIPTION
**Pull request by gintama91**
### Description

arc refer 
https://github.com/scarpe-team/scarpe/blob/2f3a881e1159ab53a655cc738397a76d84870a76/docs/static/manual.md?plain=1#L1660

### Image(if needed, helps for a faster review)

image

### Checklist

- [ ] Run tests locally
- [x] Run linter(check for linter errors)

### CODE REVIEW

File: examples/shapes/arc.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Shoes.app(height:600) do
+  arc 400, 0, 120, 100, 175, 175
+  arc 50, 50, 120, 100, 60, 225
```
Comment (Schwad) [inline examples/shapes/arc.rb:3]: Nice! I like having multiples in example tests

Can we also test it renders inside something like a stack?

And I haven't looked at the main manual but does arc support color or stroke as well?

Comment (Schwad) [reply examples/shapes/arc.rb:3]: (I would've gone through the manual link but on mobile at the second)

Comment (gintama91) [reply examples/shapes/arc.rb:3]: it does support color and stroke but more like

```
shoes.app do

fill "color"
stroke "color"
arc...

end
```

Comment (gintama91) [reply examples/shapes/arc.rb:3]: so its not something inside arc i assume

Comment (gintama91) [reply examples/shapes/arc.rb:3]: oh yeah i will check abt tht stack thing

Comment (gintama91) [reply examples/shapes/arc.rb:3]: yeah it does , should i update example with stack?

**Comment by noahgibbs** [comment]
The current test failure here *looks* related to the PR, not like a transient one from CI problems.

**Comment by gintama91** [comment]
fixd

**Review by noahgibbs — APPROVED** [review]

**Pull request merged**